yb-voyager/cmd/importData.go

```diff
@@ -1005,9 +1005,26 @@ func setTargetSchema(conn *pgx.Conn) {
 		// No need to set schema if importing in the default schema.
 		return
 	}
+	target.Schema = strings.ToLower(target.Schema)
+	checkSchemaExistsQuery := fmt.Sprintf("SELECT count(schema_name) FROM information_schema.schemata WHERE schema_name = '%s'", target.Schema)
+	rows, err := conn.Query(context.Background(), checkSchemaExistsQuery)
```

Comment:
try using `QueryRow()` function instead of `Query()`. Already used at multiple places in code
it will reduce the code